### ISSUE TITLE
When serving static files the name of resources is "null"

### ISSUE THREAD

**Issue by cgueret** [bbcarchdev/quilt#45]
Quilt shows the title "null" for every resource loaded from static files using the file engine

**Comment by nevali** [comment]
What do you mean by the name? The human-readable title used by the HTML templates? If so, it's because the HTML serialiser currently only recognises a limited set of predicates for titles.

Note that the data served by http://shakespeare.acropolis.org.uk uses the `file` engine, and doesn't seem to exhibit the problem.

**Comment by cgueret** [comment]
Yup, that title. This bug shows up for all the files in https://git.bbcarchdev.net/oss/quilt/tree/develop/docker/cucumber . They probably all use a different predicate that those used for http://shakespeare.acropolis.org.uk/
